draft-ietf-quic-transport.md

```diff
@@ -5578,9 +5578,9 @@ value.
 
 An endpoint that receives a NEW_CONNECTION_ID frame with a sequence number
 smaller than the Retire Prior To field of a previously received
-NEW_CONNECTION_ID frame MUST immediately send a corresponding
-RETIRE_CONNECTION_ID frame that retires the newly received connection ID.
-
+NEW_CONNECTION_ID frame MUST send a corresponding RETIRE_CONNECTION_ID frame
```

Comment:
"promptly send"? Though that's almost the same as immediately.

Reply:
Yeah, this is fine as is.